classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label, :students
+
+  def initialize(lebal)
+    @lebal = lebal
```

Comment:
- [ ] In these two lines, the word `label` is misspelled, it has been written as `lebal` instead. Please check on the typo so as to match the attribute_accesor for label..

Reply:
You are welcome 🥳